src/CMakeLists.txt

```diff
@@ -13,6 +13,46 @@ set(rime_core_module_src
   ${rime_api_src}
   ${rime_base_src}
   ${rime_config_src})
+
+# add rc info for windows MSVC build
+if(MSVC)
+  # check if HEAD on master
```

Comment:
discussion: I wonder if we can include a separate cmake file, since it's a lot of code.